src/components/Header/styled.ts

```diff
@@ -1,21 +1,29 @@
 import styled from 'styled-components'
 import Toggle from '../Toggle'
+import Svg from '../../assets/header.svg'
 
 export const Wrapper = styled.header`
+	align-content: start;
+	background-image: url(${Svg});
+	background-repeat: no-repeat;
+	background-size: cover;
 	display: grid;
 	grid-template-columns: 1fr 3fr 1fr;
-	margin: 1em 0;
+	height: 45em;
+	min-width: max-content;
+	width: 100%;
```

Comment:
I wouldn't force this width, because on smaller devices it creates horizontal scrollbar, which is not good responsive design practise. :sweat_smile:
```suggestion
```